indico/htdocs/sass/modules/_surveys.scss

```diff
@@ -28,6 +28,17 @@
     padding: 1em;
 }
 
+%survey-questionnaire-handle {
+    @include icon-before('icon-grid2');
```

Comment:
changed this to use a single handle class and js-classes. also removed some duplicate html from the two macros rendering text/question blocks